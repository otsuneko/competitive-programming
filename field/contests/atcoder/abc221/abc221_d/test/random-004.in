7
3 6
7 8
2 2
3 6
8 4
4 6
10 10
