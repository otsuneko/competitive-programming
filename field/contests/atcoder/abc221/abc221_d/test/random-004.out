6 3 7 1 1 0 0
